### PR TITLE
upgrade chartIQ to version v6.2.2

### PR DESCRIPTION
**Pull request by shayanbinary**

### CODE REVIEW

File: src/store/ChartStore.js [hunk 1]
```diff
@@ -446,6 +446,7 @@ class ChartStore {
     setMainSeriesDisplay(name) {
         // Set display name of main series (to be shown in crosshair tooltip)
         this.stxx.chart.seriesRenderers._main_series.seriesParams[0].display = name;
+        this.stxx.chart.seriesRenderers._main_series.seriesParams[0].field = 'Close';
     }
```
Comment (shayanbinary) [inline src/store/ChartStore.js]: @brucebinary After some hours working on this, and according to ChartIQ Team, I use this temporary solution for crosshair

Comment (brucebinary) [reply src/store/ChartStore.js]: @shayanbinary as long as it works. >_< if the solution is temporary, do add a `//TODO...` to indicate that it needs to be removed in the future.

File: src/store/ChartStore.js [hunk 2]
```diff
@@ -446,6 +429,7 @@ class ChartStore {
     setMainSeriesDisplay(name) {
         // Set display name of main series (to be shown in crosshair tooltip)
         this.stxx.chart.seriesRenderers._main_series.seriesParams[0].display = name;
+        // TODO
```
Comment (brucebinary) [inline src/store/ChartStore.js]: @shayanbinary what's this.

Comment (shayanbinary) [reply src/store/ChartStore.js]: @brucebinary :D a todo so that we remember, we should change this line of code

**Pull request merged**